R/compute_scale.R

```diff
@@ -97,19 +97,16 @@ compute_scale <- function(source,
   assert_integer_scalar(min_n, subset = "positive")
 
   # Computation
-  if (sum(!is.na(source)) >= min_n) { # nolint: undesirable_function_linter
-    target <- mean(source, na.rm = TRUE) # nolint: undesirable_function_linter
+  if (sum(!is.na(source)) >= min_n) {
+    target <- mean(source, na.rm = TRUE)
 
     if (!is.null(source_range) && !is.null(target_range)) {
-      scale_constant <- min(target_range) - min(source_range)
-      scale_coefficient <- (max(target_range) - min(target_range)) /
-        (max(source_range) - min(source_range))
-
-      target <- (target + scale_constant) * scale_coefficient
-
-      if (flip_direction == TRUE) {
-        target <- max(target_range) - target
-      }
+      target <- transform_range(
+        target,
+        source_range = source_range,
+        target_range = target_range,
+        flip_direction = flip_direction
```

Comment:
Should compute_scale inherit transform_range's documentation?

Reply:
The descriptions of the arguments differ. Thus they can't be inherited.

Reply:
but isn't this used in `compute_scale()` or am I missing something?